NitroxModel/Helper/Validate.cs

```diff
@@ -68,6 +68,21 @@ public static void IsFalse(bool b)
             }
         }
 
+        public static void String(string s)
+        {
+            if (!string.IsNullOrWhiteSpace(s))
+            {
+                return;
+            }
+            
+            Optional<string> paramName = GetParameterName<string>();
+            if (paramName.HasValue)
+            {
+                throw new ArgumentException("String must not be null of whitespace", paramName.Value);
+            }
+            throw new ArgumentException("String must not be null of whitespace");
+        }
```

Comment:
Is this whole extra logic needed ? I think this can be simplified

Reply:
Yup, ArgumentException checks for null so can just pass null into it to remove if statement.